docs/quick-starts/bandit-quickstart.md

```diff
@@ -0,0 +1,201 @@
+      " subject_numeric_attributes, subject_categorical_attributes," +
+      " action_numeric_attributes, action_categorical_attributes) " +
+      "SELECT ?, ?, ?, ?," +
+      " ?, ?, ?," +
+      " parse_json(?), parse_json(?)," +
+      " parse_json(?), parse_json(?)";
+
+    try (PreparedStatement statement = snowflakeConnection.prepareStatement(sql)) {
+      statement.setTimestamp(1, new Timestamp(logData.timestamp.getTime()));
+      statement.setString(2, logData.experiment);
+      statement.setString(3, logData.banditKey);
+      statement.setString(4, logData.subject);
+      statement.setString(5, logData.action);
+      statement.setDouble(6, logData.actionProbability);
+      statement.setString(7, logData.modelVersion);
+      if (logData.subjectNumericAttributes == null) {
+        statement.setNull(8, Types.NULL);
+      } else {
+        statement.setString(8, EppoAttributes.serializeNonNullAttributesToJSONString(logData.subjectNumericAttributes));
+      }
+      if (logData.subjectCategoricalAttributes == null) {
+        statement.setNull(9, Types.NULL);
+      } else {
+        statement.setString(9, EppoAttributes.serializeNonNullAttributesToJSONString(logData.subjectCategoricalAttributes));
+      }
+      if (logData.actionNumericAttributes == null) {
+        statement.setNull(10, Types.NULL);
+      } else {
+        statement.setString(10, EppoAttributes.serializeNonNullAttributesToJSONString(logData.actionNumericAttributes));
+      }
+      if (logData.actionNumericAttributes == null) {
+        statement.setNull(11, Types.NULL);
+      } else {
+        statement.setString(11, EppoAttributes.serializeNonNullAttributesToJSONString(logData.actionCategoricalAttributes));
+      }
+
+      statement.executeUpdate();
+    } catch (SQLException e) {
+      throw new RuntimeException("Unable to log bandit assignment "+e.getMessage(), e);
+    }
+})
+```
+
+Note the name of this table in your data warehouse. We will use it in the next step.
+
+### 3. Create a bandit
+
+Start by creating a Bandit for the new page from the "Create" button on in the Configuration section:
+
+![Create menu with Bandit selected](/img/bandit/bandit-qs-0.png)
+
+Give the Bandit a descriptive human readable name. Select the entity and input the name of table in your warehouse where you are logging bandit actions from Step 2. Choose a metric to optimize the Bandit for and how long you want the lookback window to be. The lookback window defines the time period the Bandit looks for changes in the optimization metric for subjects who were assigned.
+
+![Setup Bandit step](/img/bandit/bandit-qs-1.png)
+
+### 4. Add variations
+To evaluate the impact of the Contextual Bandit, we run an experiment with exactly two variations: the bandit and a control. This control variation should be the "status quo" that you implement on your end. A simple example is a fixed action.
+
+![Add variations step](/img/bandit/bandit-qs-2.png)
+
+### 5. Initialize the SDK
```

Comment:
I went back and forth on it. Initialize the SDK is also step 5 in the FF quickstart. But let me try moving it up

Reply:
Actually, I think it's fine where it is. I want to minimize the switching between the application and code. To initialize the SDK the logger should be defined. But we also need the table for the logging data to create the bandit. That's why I put logging so high up.

After the logger is defined, the user then creates the bandit, which is needed to embed the bandit. Then the user does SDK initialization and embeds the bandit back to back.

It's definitely tricky. The need for the warehouse table to create the bandit is obstacle to what I think could be a more straightforward flow

Reply:
That is a good point. Doesn't need to be done now, but we could have the logger example be creating a variable (rather than inlined in the client initialization) that is then passed to the initialization like the example in my other comment